cli/tests/test_comments.rs

```diff
@@ -0,0 +1,20 @@
+use crate::{TestCli, TestSource};
+
+const SAMPLE_BASIC: &str = include_str!("./samples/basic.jsonl");
+
+#[test]
+fn test_upload_comments() {
+    let cli = TestCli::get();
+    let source = TestSource::new();
+
+    let output = cli.run_with_stdin(
+        &[
+            "create",
+            "comments",
+            "--allow-duplicates",
+            &format!("--source={}", source.identifier()),
```

Comment:
Yep I think it would work too, though I personally prefer to have the pair on one line rather than 

```
"--source",
source.indentifier()
```

(I used to logically group flags and their values on one line, but automatic formatting has taken that joy away from me!)

Reply:
Ah, fair enough